simulators/portal-interop/src/main.rs

```diff
@@ -54,155 +57,124 @@ async fn run_suite(host: Simulation, suite: Suite) {
+            last_header = history_content.clone();
+        }
+        let (content_type, block_number, test_data) =
+            if let HistoryContentValue::BlockHeaderWithProof(header_with_proof) = &last_header.1 {
+                match &history_content.0 {
+                    HistoryContentKey::BlockHeaderWithProof(_) => (
+                        "Block Header".to_string(),
+                        header_with_proof.header.number,
+                        vec![content_pair_to_string_pair(last_header.clone())],
+                    ),
+                    HistoryContentKey::BlockBody(_) => (
+                        "Block Body".to_string(),
+                        header_with_proof.header.number,
+                        vec![
+                            content_pair_to_string_pair(last_header.clone()),
+                            content_pair_to_string_pair(history_content),
+                        ],
+                    ),
+                    HistoryContentKey::BlockReceipts(_) => (
+                        "Block Receipt".to_string(),
+                        header_with_proof.header.number,
+                        vec![
+                            content_pair_to_string_pair(last_header.clone()),
+                            content_pair_to_string_pair(history_content),
+                        ],
+                    ),
+                    HistoryContentKey::EpochAccumulator(_) => (
+                        "Epoch Accumulator".to_string(),
+                        header_with_proof.header.number,
+                        vec![],
+                    ),
+                }
+            } else {
+                unreachable!("History test dated is formatted incorrectly")
+            };
+        result.push(ProcessedContent {
+            content_type,
+            block_number,
+            test_data,
+        })
+    }
+    result
+}
+
+fn get_flair(block_number: u64) -> String {
+    if block_number > 17034870 {
+        " (post-shanghai)".to_string()
+    } else if block_number > 15537394 {
+        " (post-merge)".to_string()
+    } else if block_number > 12965000 {
+        " (post-london)".to_string()
+    } else if block_number > 12244000 {
+        " (post-berlin)".to_string()
+    } else if block_number > 9069000 {
+        " (post-istanbul)".to_string()
+    } else if block_number > 7280000 {
+        " (post-constantinople)".to_string()
+    } else if block_number > 4370000 {
+        " (post-byzantium)".to_string()
+    } else if block_number > 1150000 {
+        " (post-homestead)".to_string()
+    } else {
+        "".to_string()
+    }
+}
+
 dyn_async! {
    async fn test_portal_interop<'a> (test: &'a mut Test, _client: Option<Client>) {
         // Get all available portal clients
         let clients = test.sim.client_types().await;
 
+        let values = std::fs::read_to_string("./test-data/test_data_collection_of_forks_blocks.yaml")
```

Comment:
Could you move this filename to a const w/ a brief comment describing what kind of data should go in the file / why one might add more data to the file / what tests are run for any data in the file

Reply:
resolved